recipes/libyuv/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibyuvConan(ConanFile):
+    name = "libyuv"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/libyuv/libyuv/"
+    description = "libyuv is an open source project that includes YUV scaling and conversion functionality."
+    topics = ["YUV", "libyuv", "google", "chromium"]
+    license = "BSD"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True}
+    requires = ("libjpeg-turbo/2.0.5")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+
+    _cmake = None
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", src=".", dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
```

Comment:
for future reference, https://chromium.googlesource.com/libyuv/libyuv/+/2664f649bc0a32e05f5bc3ff4d12134e41aa08c4/CMakeLists.txt#14

CMake components

static is `yuv`
shared is `yuv_shared`

no namespace

Reply:
Any example here? I'm not very familiar with this.

Reply:
https://github.com/conan-io/conan-center-index/blob/d435402a2a83c8f84b19e007a2766a9629ba4986/recipes/hdf4/all/conanfile.py#L110-L112

We'd appreciate any extra effort ❤️

Reply:
libyuv static and stared library output name is same. Does this cause conflict?

Reply:
Nope!

Reply:
I found It is a little difficult for me to resolv this conversation after several tries. I wonder if you could help me with this?

Reply:
This is not required for an approval, I just left it here for a note

```suggestion
    def package_info(self):
    #FIXME: Missing CMake components: https://chromium.googlesource.com/libyuv/libyuv/+/2664f649bc0a32e05f5bc3ff4d12134e41aa08c4/CMakeLists.txt#14
```